Resources/Prototypes/_DEN/Mobs/Customization/Markings/vulpkaninmarks.yml

```
@@ -0,0 +1,53 @@
+﻿- type: marking
+  id: NeckChestfluff
+  bodyPart: Chest
+  markingCategory: Chest
+  speciesRestriction: [Vulpkanin]
+  sprites:
+  - sprite: _DEN/Mobs/Customization/Vulpkanin/vulpkanin.rsi
+    state: chestfloof
+
+
+- type: marking
+  id: TailBrushtip
+  bodyPart: Tail
+  markingCategory: Tail
+  speciesRestriction: [Vulpkanin]
```

Give the same restrictions as the similiar Eastern Dragon tail from Skyrat.
```suggestion
  speciesRestriction: [Vulpkanin, Reptilian, SlimePerson, IPC, Rodentia, Vulpkanin, Felinid, Tajaran, Human, Oni, Kitsune] # Floof - Anthro Marking Restrictions
```